spec/unit/matrixrtc/mocks.ts

```diff
@@ -18,27 +18,31 @@ import { EventType, MatrixEvent, Room } from "../../../src";
 import { CallMembershipData } from "../../../src/matrixrtc/CallMembership";
 import { randomString } from "../../../src/randomstring";
 
-export function makeMockRoom(
-    memberships: CallMembershipData[],
-    getLocalAge: (() => number) | undefined = undefined,
-): Room {
+export function makeMockRoom(memberships: CallMembershipData[], localAge: number | null = null): Room {
     const roomId = randomString(8);
+    // If makeMockRoom is used instead of one of the other functions in this file,
+    // we store it so that subsequent calls calculate the expiration based on it using
+    // Date.now(). So, jest.advanceTimersByTime() can be used to simulate the passage of time.
+
+    // The actual time of the underlying mocked events will be: localTimestamp ?? localAge ?? 0
```

Comment:
> we store it

what is "it" in this comment? `makeMockRoom`?

I'd suggest rewording this whole comment for clarity.

```suggestion
    // Replace the `localAge` with a `localTimestamp`, so that `jest.advanceTimersByTime()` can be used
    // to simulate the passage of time and test expiry.
```

Reply:
I removed the comment. With the last [commit](https://github.com/matrix-org/matrix-js-sdk/pull/3900/commits/54b71bb729090a7534a54bd6dbfdc3e3d738d66a), we store the whole return value outside the mocked method so it wont change anyways. (and its quiet clear even without comment)